### DISCUSSION TITLE
WindiCSS - Not maintaining anymore?

### DISCUSSION THREAD

**Discussion by gtu-myowin** [windicss/windicss#625] [General]
I am about to try WindiCSS for my new projects and I went ahead to install using the CLI but when I ran `windicss -h` command I got `TypeError: Cannot read property 'Glob' of undefined` error. So I search in the Issues tab for the solution and found out some people have already asked about this but without any answer.

I know that WindiCSS is solely community base with a handful of core contributors which is why I would like to try Windi but **with bugs and no active support** make me hesitate to give Windi a try and led me to the question "Is Windi still being maintained?".

**Comment by antfu** [comment]
Yes, it's still been actively maintained. As you probably see we have released a new minor one week ago.

I am sorry to know the issue you have faced is not been solved yet. But given that this is a community project with very little sponsorship income, no one is working full time on it. Maintainers have things to prioritize, so maybe your issue does get enough attention from the maintainers, unfortunately (e.g. myself don't use the CLI so I have very little incentive to fix it). Note this might also apply to millions of issue reports in the open-source community. I think why open source is still awesome is that it's open for contributions. If you really need it to be fixed, maybe it could be a good chance for you to take a look and even send a PR (and I believe in your case it's fairly straightforward).

**Reply by gtu-myowin** [reply]
Thank you for your response. As much as I would like to, I still need a lot of knowledge to do the contribution but I will give it a try. Keep up the good work!
